src/renderer/features/balances/subscription/lib/balance-sub-utils.ts

```diff
@@ -41,15 +41,16 @@ function getAccountsToSubscribe(
 
     if (!proxyWallet || !proxyAccount) return [firstAccount];
 
-    return [firstAccount, ...getAccountsToSubscribe(proxyWallet, wallets, [proxyAccount], accounts)];
+    return [firstAccount, ...getSiblingAccounts(proxyWallet, wallets, [proxyAccount], accounts)];
   }
 
   return walletAccounts;
 }
 
-function getNewAccounts(
-  subAccounts: SubAccounts,
+function formSubAccounts(
+  walletId: ID,
```

Comment:
This way all accounts are connected to one wallet
Otherwise we end up having 2 wallets for 1 proxied wallet (proxied wallet + proxy wallet)